CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 - Listen to `ViewModeAvailabilityChanged` event and toggle visibility of `FullscreenToggleButton` and `PictureInPictureToggleButton` accordingly
 
+### Changed
+- TypeScript update to 3.9.6
```

Comment:
Isn't this more of an `## Internal` change?

Reply:
I thought so too, but there are no `## Internal` entries in this Changelog (maybe because it is public?) and the last TS upgrade was also in `## Changed`

Reply:
Sry, didn't look at the repository name 🙈